<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="icon">
       <iconset resource="../../resources/resource.qrc">
        <normaloff>:/icons_black/icons_black/toggle_off_black.png</normaloff>:/icons_black/icons_black/toggle_off_black.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Program</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkLockWhenMinimized">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Lock when minimized</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="checkShowSubGroupElements">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Show elements from subgroup</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="checkHideFromTaskbar">
         <property name="text">
          <string>Hide from taskbar when minimized</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkLockAfterTime">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Lock when inactive (seconds)</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QCheckBox" name="checkOpenLastDbOnStartUp">
         <property name="text">
          <string>Open last password container at start-up</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="textInactivityTime">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkMinimizeWhenLocking">
         <property name="text">
          <string>Minimize to tray when locked</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="3">
        <widget class="QCheckBox" name="checkAutoSave">
         <property name="text">
          <string>Automatically save when entry has been edited</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(199, 199, 199);</string>
         </property>
         <property name="text">
          <string>Open &amp; Save</string>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(199, 199, 199);</string>
         </property>
         <property name="text">
          <string>Security</string>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(199, 199, 199);</string>
         </property>
         <property name="text">
          <string>Look &amp; Feel</string>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QCheckBox" name="checkSecureDesktop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Secure Desktop</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="buttonChangePassword">
         <property name="text">
          <string>Change password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_shortcut">
      <attribute name="icon">
       <iconset resource="../../resources/resource.qrc">
        <normaloff>:/icons_black/icons_black/ctrl_black.png</normaloff>:/icons_black/icons_black/ctrl_black.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Shortcuts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="2">
        <widget class="QKeySequenceEdit" name="keySequenceBrintToFront"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_12">
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimizes or restors the main window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Show window:</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(199, 199, 199);</string>
         </property>
         <property name="text">
          <string>Program shortcuts</string>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QKeySequenceEdit" name="keySequenceOpenUrl"/>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="toolButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/external_link2_black.png</normaloff>:/icons_black/icons_black/external_link2_black.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QKeySequenceEdit" name="keySequenceCopyPassword"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Open URL:</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QKeySequenceEdit" name="keySequenceCopyUsername"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Copy password:</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="toolButton_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/user_group_man2_black.png</normaloff>:/icons_black/icons_black/user_group_man2_black.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="toolButton_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/copy2_black.png</normaloff>:/icons_black/icons_black/copy2_black.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QToolButton" name="toolButton_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/enter2_black.png</normaloff>:/icons_black/icons_black/enter2_black.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Copy username:</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QToolButton" name="toolButton_5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/save2_black.png</normaloff>:/icons_black/icons_black/save2_black.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QKeySequenceEdit" name="keySequencePerformAutoType"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Save:</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Perform autotype:</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QKeySequenceEdit" name="keySequenceSave"/>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(199, 199, 199);</string>
         </property>
         <property name="text">
          <string>Global shortcuts</string>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QToolButton" name="toolButton_6">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/restore_window2_black.png</normaloff>:/icons_black/icons_black/restore_window2_black.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../../resources/resource.qrc">
        <normaloff>:/icons_black/icons_black/puzzle_black.png</normaloff>:/icons_black/icons_black/puzzle_black.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Browser plugin</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="QLineEdit" name="textCommunicationAccessToken">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Access token...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Token:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(199, 199, 199);</string>
         </property>
         <property name="text">
          <string>Plugin settings</string>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Port:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="textPortNumber">
         <property name="minimum">
          <number>1234</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QCheckBox" name="checkEnableCommunicationPort">
         <property name="text">
          <string>Enable browser plugin</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Help:&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;1) Enable communication port.&lt;br/&gt;2) Install the myPasswords browser plugin.&lt;br/&gt;3) Open the settings panel of the browser plugin.&lt;br/&gt;4) Copy the Access in the textfield above to the token field of the plugin.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_language">
      <attribute name="icon">
       <iconset resource="../../resources/resource.qrc">
        <normaloff>:/icons_black/icons_black/globe_black.png</normaloff>:/icons_black/icons_black/globe_black.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Language</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listLanguage"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
